kimchi/src/prover.rs

```diff
@@ -85,11 +85,26 @@ impl<G: CommitmentCurve> ProverProof<G>
 where
     G::BaseField: PrimeField,
 {
-    /// This function constructs prover's zk-proof from the witness & the ProverIndex against SRS instance
+    /// This function constructs prover's non-recursive zk-proof from the witness & the ProverIndex against SRS instance
     ///     witness: computation witness
     ///     index: ProverIndex
     ///     RETURN: prover's zk-proof
```

Comment:
I'd be nice to have a nicer doc comment (that matches how the rust stdlib is documented for example, instead of listing the arguments like that). Also, I suggest not mentioning non-recursion here, and only mentioning recursion in the doc of the other function